app/src/main/java/org/tokend/template/features/kyc/model/KycForm.kt

```diff
@@ -10,12 +10,13 @@ import org.tokend.sdk.api.blobs.model.Blob
  */
 sealed class KycForm(
         @SerializedName("documents")
-        var documents: MutableMap<String, RemoteFile>? = mutableMapOf()
+        var documents: MutableMap<String, RemoteFile>? = mutableMapOf(),
+        var role: String? = null
```

Comment:
- Name it `roleKey` because it is not role itself
- Why `var`?
- Why optional?
- It must not be a constructor argument, form role key can't be selected by user. Make it an abstract property and then add overrides to each type. For empty throw `IllegalStateException` in getter saying that you can't use empty form to change role